x-pack/plugins/security/server/authentication/providers/saml.ts

```diff
@@ -389,7 +393,9 @@ export class SAMLAuthenticationProvider extends BaseAuthenticationProvider {
       });
     } catch (err) {
       this.logger.debug(`Failed to perform IdP initiated local logout: ${err.message}`);
-      return AuthenticationResult.failed(err);
+
+      // HACK: WE SHOULD UNCOMMENT THIS LINE ONCE https://github.com/elastic/elasticsearch/issues/53323 IS FIXED!
```

Comment:
__note:__ we can keep this if we decide to fix the issue without ES part in https://github.com/elastic/elasticsearch/issues/53323.